src/components/Comments.vue

```diff
@@ -4,57 +4,60 @@
       <i class="el-icon-s-comment"></i>
       {{ $t('comments.title') }}
     </div>
-    <div
-      v-for="(comment, index) in comments"
-      :key="index"
-      class="py-3 grid grid-cols-3 sm:grid-cols-4 md:grid-cols-8 border-b"
-      :class="index === comments.length - 1 ? 'border-none pb-0' : ''"
-    >
-      <div id="user-area" class="h-full col-span-1">
-        <img
-          class="w-full rounded-full"
-          :src="getAvatar(get(comment, 'relationships.creator[0].avatar'))"
-        />
-      </div>
+    <transition-group name="fade" tag="div">
       <div
-        id="comment-area"
-        class="pl-4 col-span-full col-start-2 place-content-start"
+        v-for="(comment, index) in comments"
+        :key="comment.id"
+        class="py-3 grid grid-cols-3 sm:grid-cols-4 md:grid-cols-8 border-b"
+        :class="index === comments.length - 1 ? 'border-none pb-0' : ''"
```

Comment:
Please prefer using `last-child` instead.
It's pure css and more performant https://tailwindcss.com/docs/hover-focus-and-other-states#last-child
Usage example `last:border-none last:pb-0`